packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.0
+
+  * Added option selectionControls
```

Comment:
Nit: Adds

See the CHANGELOG style guide linked from the PR checklist.

Reply:
I didn't understand how to change. I use the current style.

## 0.6.8

  * Added option paddingBuilders
  
## 0.6.7

 * Fix `unnecessary_import` lint errors.
 * Added option pPadding
 * Added options h1Padding - h6Padding

Reply:
> I didn't understand how to change.

You need to change "Added" to "Adds"

> I use the current style.

You need to follow the store guide I referenced, which is newer than the existing entries.

Reply:
Thanks. Updated